kore/src/Kore/ModelChecker/Bounded.hs

```diff
@@ -0,0 +1,190 @@
+{-|
+Module      : Kore.ModelChecker.Bounded
```

Comment:
The `Module:` field is redundant, so we don't include it anymore.

Reply:
done